lib/constants.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
+const { CLUSTER_BASE_URL } = require('./utils/configurations');
+
+const clusters = {
+  new: 'new',
+  top: 'top'
+};
+
+const collectionPaths = {
+  TOP_FREE: { collection: 'topselling_free', path: `${CLUSTER_BASE_URL}/${clusters.top}` },
```

Comment:
I find the code in this module very hard to follow: 

1. It's hard to tell what the final urls look like by looking at the code here and the code where these constants are used
2. the fact that you need to reduce to export these and then process on the other side to extrac collection + path looks suspicious.
3. this collection+path fields here also seem more than necessary. If you really need to map each category constant to the cluster type (top vs new), then why not

```js
 TOP_FREE: { collection: 'topselling_free', cluster: 'top' },
...
NEW_FREE: { collection: 'topselling_new_free', cluster: 'new' },
... 
```
?